src/display/annotation_layer.js

```diff
@@ -490,9 +493,18 @@ var PopupElement = (function PopupElementClosure() {
       var wrapper = document.createElement('div');
       wrapper.className = 'popupWrapper';
 
-      var popup = this.popup = document.createElement('div');
+      // For Popup annotations we hide the entire section, but for Text
+      // annotations we do not want the image to be hidden too, so there
+      // we just hide the wrapper.
+      if (this.hideContainer) {
+        this.hideElement = this.container;
+      } else {
+        this.hideElement = wrapper;
+      }
```

Comment:
If you wanted to, this could probably be made a one-liner, e.g.
`this.hideElement = this.hideContainer ? this.container : wrapper;`